deployments/ansible/roles/collector/tasks/win_fluentd_install.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Set fluentd package sources
+  ansible.builtin.set_fact:
+    td_agent_base_url: https://packages.treasuredata.com
+    win_fluentd_msi: td-agent-{{td_agent_version}}-x64.msi
+
+- name: Download fluentd
+  ansible.windows.win_get_url:
+    url: "{{td_agent_base_url}}/4/windows/{{win_fluentd_msi}}"
+    dest: "%TEMP%"
+  register: fluentd_msi
+
+- name: Install Fluentd on Windows
+  ansible.windows.win_package:
+    path: "{{fluentd_msi.dest}}"
+    state: present
```

Comment:
Probably `notify: "restart windows fluentdwinsvc"` should be added as for otel collector